### ISSUE TITLE
Comments For Better Code Documentation

### ISSUE THREAD

**Issue by Moksh45** [gxrsti/alert#36]
#18

**Comment by Moksh45** [comment]
Hey @gxrsti, I remember you suggested using JSDoc-style comments. I made some changes, and now the comments look like this. Could you please review them and let me know if you're satisfied? 
```
/**
 * Toaster component for displaying toast notifications.
 * @param {object} ToasterProps - Props for configuring the toaster.
 *   @property {string} position - Position to display the toaster (default: 'bottom-right').
 *   @property {number} duration - Duration to display each toast (default: 3000ms).
 *   @property {boolean} reverse - Whether to reverse the order of toasts (default: false).
 *   @property {string} theme - Theme for the toasts (default: 'light').
 *   @property {object} style - Custom CSS style for the toaster.
 *   @property {string} className - Custom CSS class for the toaster.
 * @returns {JSX.Element} - A component for displaying toast notifications.
 */
```

**Comment by gxrsti** [comment]
Hey @Moksh45, yeah thats what i am looking for!

**Comment by Moksh45** [comment]
i'm submitting two pull requests for this issue, as I need to fulfill my Hacktoberfest PR quota

**Comment by Moksh45** [comment]
hmm! OK

**Comment by Moksh45** [comment]
Do I need to write documentation for the [website](https://github.com/Moksh45/alert/tree/master/website) folder as well?

**Comment by gxrsti** [comment]
Resolved by #37
